specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/integrationRuntime.json

```diff
@@ -1323,6 +1323,55 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/integrationRuntimes/{integrationRuntimeName}/stop/8A2431BC39114B709A447C12F99E3BB4": {
```

Comment:
We should not be including stop verb in get operation.

Reply:
Work item has Stop for Get operation
Missing Operation
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.SYNAPSE/WORKSPACES/INTEGRATIONRUNTIMES/STOP/8A2431BC39114B709A447C12F99E3BB4
![image](https://user-images.githubusercontent.com/109329038/191443412-27ff6c29-6304-4a8f-b899-9a5a4225e275.png)